docs/index.bs

```diff
@@ -3065,6 +3065,8 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
               :: Return |serviceWorker|'s [=service worker/script url=].
               : The [=environment settings object/origin=]
               :: Return its registering [=/service worker client=]'s [=environment settings object/origin=].
+              : The [=environment settings object/cross site ancestry=]
+              :: Return its registering [=/service worker client=]'s [=environment settings object/cross site ancestry=].
```

Comment:
I assume that `cross site ancestry` is used for noticing 3rd party site ancestry to avoid unexpected cookie access there.  Then, I wonder if it is valid to use the registration time client for decision making.
Assuming that the registration has been done by Origin A, but actual usage is fetching from ServiceWorker-controlled iframe Origin A inside Origin B.  For that case, I guess fetching inside iframe Origin A can be captured by ServiceWorker registered by the Origin A (not iframe's) if the storage partitioning is disabled.  Then, the iframe itself might be `cross site ancestry` == true but the ServiceWorker intercepting the request might be `cross site ancestry` == false.  Is it intended behavior?

Reply:
Okay, that makes sense! Thank you for clearing that up. 

I think this actually works fine in the case that storage partitioning is enabled because the cross-site ancestry will be the same between the frame initiating the fetch and the worker handling it.

Currently there are some plans to specify all of the storage partitioning implementation changes, particularly from @kyraseevers. 

The spec works as-is because of some hand-waving in Fetch in how `SameSite` is handled. Would it be reasonable to add an inline issue somewhere in this spec pointing out that if storage partitioning is disabled, the invocation of fetch should be done with a client with `cross site ancestry` matching the initiating client?

Reply:
Thanks for the reply.

Could you please add a note mentioning that `cross-site ancestry` may not be set correctly if Storage Partitioning is disabled? I think we should strongly recommend enabling it for any algorithm that relies on `cross-site ancestry`.

Other than that, looks good to me.

Reply:
I've added a note in the list. Broken references should resolve once we merge https://github.com/whatwg/html/pull/11133.

Thank you for the feedback!